plugins/woocommerce/changelog/48383-fix-shipping_method_index

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+fix: set correct shipping_method indexes in cart  </details>  <details>  <summary>Changelog Entry Comment</summary>
```

Comment:
I think something went wrong with the changelog generation tool here, as we've got various bits of extra markup. Tweak:

```suggestion
Set the correct shipping method indexes in the cart/cart-shipping template.
```